packages/@react-spectrum/s2/style/spectrum-theme.ts

```diff
@@ -599,65 +659,33 @@ export const style = createTheme({
       },
       code: 'source-code-pro, "Source Code Pro", Monaco, monospace'
     },
-    fontSize: {
-      // The default font size scale is for use within UI components.
-      'ui-xs': fontSizeToken('font-size-50'),
-      'ui-sm': fontSizeToken('font-size-75'),
-      ui: fontSizeToken('font-size-100'),
-      'ui-lg': fontSizeToken('font-size-200'),
-      'ui-xl': fontSizeToken('font-size-300'),
-      'ui-2xl': fontSizeToken('font-size-400'),
-      'ui-3xl': fontSizeToken('font-size-500'),
-
-      control: {
-        default: fontSizeToken('font-size-100'),
-        size: {
-          XS: fontSizeToken('font-size-50'),
-          S: fontSizeToken('font-size-75'),
-          L: fontSizeToken('font-size-200'),
-          XL: fontSizeToken('font-size-300')
-        }
-      },
+    fontSize,
+    fontWeight: createMappedProperty((value, property) => {
+      if (property === 'fontWeight') {
+        return {
+          // Set font-variation-settings in addition to font-weight to work around typekit issue.
+          fontVariationSettings: value === 'inherit' ? 'inherit' : `"wght" ${value}`,
+          fontWeight: value as any,
+          fontSynthesisWeight: 'none'
```

Comment:
This property prevents the browser from trying to synthesize bold font weights. For the new font, it think the only weight available is 100 so tries to thicken the letters. We don't need that since we are using `font-variation-settings`.